src/Uno.UI/UI/Xaml/Controls/TextBlock/TextBlock.iOS.cs

```diff
@@ -265,6 +266,14 @@ private UIStringAttributes GetAttributes()
 				attributes.KerningAdjustment = (CharacterSpacing / 1000f) * 12;
 			}
 
+			// Foreground checks should be kept at the end since we **may** use the attributes to calculate text size
+			// for gradient brushes.
+			// TODO: Support other brushes (e.g. gradients):
+			if (Brush.TryGetColorWithOpacity(Foreground, out var color))
```

Comment:
Do we no longer have to use Colors.Transparent as a defaultColor when attempting to get the color from the brush?

Reply:
Done in 1da1acc